docs/quick-start.md

```diff
@@ -27,15 +27,15 @@ Follow the steps below to get up and running for the configuration backup elemen
 
 4. Create Configuration Removals and Replacements.
 
-    1. [Config Removals](./navigating-backup#config-removals)
-    2. [Config Replacements](./navigating-backup#config-replacements)
+    1. [Config Removals](./navigating-backup.md#config-removals)
```

Comment:
fyi, these links work without it, but I see some have .md so for consistency sake, makes sense.

Reply:
I changed them because I found them broken while clicking on them